tests/api-tests/utils.ts

```diff
@@ -17,21 +17,24 @@ export const apiTestConfig = (
 });
 
 const unpackErrors = (errors: readonly any[] | undefined) =>
-  (errors || []).map(({ locations, extensions: { exception, ...extensions }, ...unpacked }) => ({
-    extensions,
-    ...unpacked,
-  }));
+  (errors || []).map(({ locations, ...unpacked }) => unpacked);
```

Comment:
This is the actual change. Everything else in this PR is in response to this change.